website/templates/include/profile/schools.mako

```diff
@@ -96,7 +95,7 @@
             </div>
 
             <div>
-                <a class="btn btn-default" data-bind="click: addContent">
+                <a class="btn btn-default" data-bind="disable, click: addContent">
```

Comment:
Disable is a knockout binding to disable the button if the validation requirements are not met.